cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/server/GameServer.java

```diff
@@ -387,17 +315,126 @@ public boolean bootstrap() throws Exception {
+     * Configure a glowstone server.
+     *
+     * @return Given back a complete server info
+     */
+    private ServerInfo configureGlowstoneServer() {
+        String motd = null;
+        int maxPlayers = 0;
+        try (InputStreamReader inputStreamReader = new InputStreamReader(Files.newInputStream(Paths.get(path + "/config/glowstone.yml")),
+                                                                         StandardCharsets.UTF_8)) {
+            Configuration configuration = ConfigurationProvider.getProvider(YamlConfiguration.class).load(inputStreamReader);
+            Configuration section = configuration.getSection("server");
+            section.set("ip", CloudNetWrapper.getInstance().getWrapperConfig().getInternalIP());
+            section.set("port", serverProcess.getMeta().getPort());
+
+            maxPlayers = section.getInt("max-players");
+            motd = section.getString("motd");
+
+            configuration.set("server", section);
+            configuration.set("console.use-jline", false);
+            try (OutputStreamWriter outputStreamWriter = new OutputStreamWriter(Files.newOutputStream(Paths.get(path + "/config/glowstone.yml")),
+                                                                                StandardCharsets.UTF_8)) {
+                ConfigurationProvider.getProvider(YamlConfiguration.class).save(configuration, outputStreamWriter);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return new ServerInfo(serverProcess.getMeta().getServiceId(),
+                              CloudNetWrapper.getInstance().getWrapperConfig().getInternalIP(),
+                              this.serverProcess.getMeta().getPort(),
+                              false,
+                              new ArrayList<>(),
+                              serverProcess.getMeta().getMemory(),
+                              motd,
+                              0,
+                              maxPlayers,
+                              ServerState.OFFLINE,
+                              this.serverProcess.getMeta().getServerConfig(),
+                              serverProcess.getMeta().getTemplate());
+    }
 
-        serverProcess.setServerStage(ServerStage.PROCESS);
-        CloudNetWrapper.getInstance().getServers().put(this.serverProcess.getMeta().getServiceId().getServerId(), this);
-        serverProcess.setServerStage(ServerStage.NET_INIT);
-        return true;
+    /**
+     * Configure a server like not a glowstone server type.
+     *
+     * @return Return the finish Server info.
+     */
+    private ServerInfo configureNoGlowstoneServer() {
+        String motd;
+        int maxPlayers;
+        Properties properties = new Properties();
+        try (InputStreamReader inputStreamReader = new InputStreamReader(Files.newInputStream(Paths.get(path + "/server.properties")))) {
```

Comment:
Use the separator-less version of `Paths.get` by specifying each directory separately
```suggestion
        try (InputStreamReader inputStreamReader = new InputStreamReader(Files.newInputStream(Paths.get(path,  "server.properties")))) {
```